builder/dockerfile/metrics.go

```diff
@@ -0,0 +1,29 @@
+package dockerfile
+
+import "github.com/docker/go-metrics"
+
+var (
+	buildsTriggered metrics.Counter
+	buildsFailed    metrics.LabeledCounter
+)
+
+func init() {
```

Comment:
Ok, as we talked on `Slack, I’ve removed the `internal` package and moved the `constants` file within the `dockerfile` package. They look like this now:
```
const (
	metricsDockerfileSyntaxError        = "dockerfile_syntax_error"
.....
)

```